packages/seacas/libraries/ioss/src/text_mesh/Iotm_TextMeshUtils.h

```diff
@@ -14,593 +14,1754 @@
 #include <vector>                                    // for vector
 #include <unordered_map>
 #include <sstream>                       // for ostringstream
+#include <iostream>
 #include <functional>
 #include <stdexcept>
+#include <numeric>
+#include <tokenize.h>
+#include <fmt/ostream.h>
+#include <strings.h>
 
 // clang-format on
 // #######################   End Clang Header Tool Managed Headers  ########################
 namespace Iotm {
-  namespace text_mesh {
-    using ErrorHandler = std::function<void(const std::ostringstream &)>;
-
-    template <class EXCEPTION> void handle_error(const std::ostringstream &message)
-    {
-      throw EXCEPTION((message).str());
+namespace text_mesh
+{
+using ErrorHandler = std::function<void(const std::ostringstream &)>;
+
+template <class EXCEPTION>
+void handle_error(const std::ostringstream &message)
+{
+  throw EXCEPTION((message).str());
+}
+
+inline void default_error_handler(const std::ostringstream &message)
+{
+  handle_error<std::logic_error>(message);
+}
+
+inline std::string strip(const std::string &inpt)
+{
+  auto start_it = inpt.begin();
+  auto end_it = inpt.rbegin();
+  while (std::isspace(*start_it)) ++start_it;
+  while (std::isspace(*end_it)) ++end_it;
+  return std::string(start_it, end_it.base());
+}
+
+inline std::vector<std::string> tokenize(const std::string &str, const std::string &separators)
+{
+  std::vector<std::string> tokens;
+  auto first = std::begin(str);
+  while (first != std::end(str)) {
+    const auto second = std::find_first_of(first, std::end(str), std::begin(separators), std::end(separators));
+    if (first != second) {
+      std::string token = strip(std::string(first, second));
+      tokens.emplace_back(token);
     }
-
-    inline void default_error_handler(const std::ostringstream &message)
-    {
-      handle_error<std::logic_error>(message);
+    if (second == std::end(str)) {
+      break;
+    }
+    first = std::next(second);
+  }
+  return tokens;
+}
+
+inline void convert_to_upper_case(std::string &str)
```

Comment:
Is it possible to use the Ioss::Utils::uppercase function.

Reply:
Please see first comment on independent portability